engine/render/src/render/render_script.cpp

```diff
@@ -1080,7 +1080,7 @@ namespace dmRender
             luaL_error(L, "Unable to create render target.");
         }
 
-        lua_pushinteger(L, render_target);
+        lua_pushnumber(L, render_target);
```

Comment:
win32 fix, integer is pushed as 32b and number as 64

Reply:
Same concern here then (re: light user data)